superchain/configs/goerli/superchain.yaml

```diff
@@ -4,4 +4,4 @@ l1:
   public_rpc: https://ethereum-goerli-rpc.allthatnode.com
   explorer: https://goerli.etherscan.io
 
-protocol_versions_addr: null # todo
+protocol_versions_addr: "0x0C24F5098774aA366827D667494e9F889f7cFc08"
```

Comment:
Just wondering if we need the string to be wrapped in `"` since none of the others are